docs/ldml/tr35.md

```diff
@@ -3170,6 +3173,41 @@ Processes sometimes encounter invalid number or date patterns, such as a number
 * For a pattern that contains a currently-invalid pattern character (applies only to date patterns, for which A-Za-z are reserved as pattern characters but not all defined as valid):
   * Produce an error (set an error code or throw an exception) when an attempt is made to create a formatter with such a pattern or to apply such a pattern to an existing formatter.
 
+## 8 <a name="Data_Size" href="#Data_Size">Data Size Reduction</a>
+Software implementations may have constrained memory requirements. The following outlines some techniques for filtering out CLDR data for a particular implementation. The exact filtering would depend on the particular requirements of the implementation in question, of course.
+
+Locale data can be _sliced_ to exclude data not needed by a particular implementation. This can be _vertical slicing_: excluding a locale and all the locales inheriting from them, or _horizontal slicing_: excluding particular types of data from all locales. For example:
+  * A vertical slice could retain only those locales used in a particular set of markets, such as EU locales.
+  * A horizontal slice could remove all data in the emoji/ directory, which are annotations for emoji and symbols.
+  
+Of course, both of these techniques can be applied.
+
+### 8.1 <a name="Vertical_Slicing" href="#Vertical_Slicing">Vertical Slicing</a>
+
+It is important to note that if a particular locale is in a vertical slice, then all of its parents should be as well, because of inheritance. This is not a factor if the data is fully resolved, as in the JSON format data.
+
+Slicing can also remove related supplemental data. For example, the likely subtags data includes a large number of languages that may not be of interest for all implementations. Here are some sample techniques can be used to filter the data so that the number of mappings is reduced:
+
+1. Only keep likely subtags data with for locales that are used in the implementation.
+  * For example, if the implementation only includes the CLDR locales at Basic coverage in [Unicode CLDR - Coverage Levels](https://cldr.unicode.org/index/cldr-spec/coverage-levels) (and locales inheriting from them), then filter out all likely subtag data that doesn’t match. 
+
+2. Filter out the locales as per #1, except also retain also those locales representing the official languages for for countries, based on data in supplementalData.xml. For a human-readable chart, see [Territory-Language Information](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/territory_language_information.html).
+
+### 8.2 <a name="Horizontal_Slicing" href="#Horizontal_Slicing">Horizontal_Slicing</a>
```

Comment:
The main reason to perform horizontal slicing is when an application wants to remove data required for powering some feature. If an app isn't performing date formatting, it can remove all transitive date formatting data, except if it is formatting currencies, it needs to retain some number formatting data. I don't understand how coverage levels play into this.

Reply:
Good point. Incorporated it, and also clarified the use of coverage information.